docs/pixmap.rst

```diff
@@ -388,9 +389,14 @@ Have a look at the :ref:`FAQ` section to see some pixmap usage "at work".
                doc.save("ocr-images.pdf")
 
 
-   ..  method:: pil_save(*args, unmultiply=False, **kwargs)
```

Comment:
This looks like it has removed the documentation for pic_save()? Was that the intention?